extensions/wikia/Forum/css/Forum.scss

```diff
@@ -287,14 +287,16 @@
 	border-bottom: 0;
 	margin: 0 10px 0 0;
 	padding-bottom: 0;
-	// DAR-1382 overwrite right padding to make place for larger forum tally
-	padding-right: 260px;
+	padding-right: 0;
 	position: relative;
 
+	h1 {
+		margin: 0;
+		word-break: normal;
+	}
+
 	.header-right {
```

Comment:
LGTM as a quick fix. Although the best idea would be to apply markup and styles from article so it becomes "social share icons" friendly.